UIs/stylesheet.css

```diff
@@ -0,0 +1,108 @@
+body {
+    margin: 0;
+    padding: 0;
+    background: url(./images/meetupPic.jpg);
+    background-size: cover;
+    background-position: center center;
+    font-family: sans-serif;
+}
+
+.login-box {
+    width: 350px;
+    height: 450px;
```

Comment:
You may need to increase the height of this to ensure the id `theForm` is found within the login-box and does not overflow. This is causing https://github.com/ViaChristopheH/questionerNew/blob/ft-UI-login-%23163992381/UIs/pages/login.html#L23 to be barely visible

Reply:
<img width="487" alt="screen shot 2019-02-18 at 15 37 55" src="https://user-images.githubusercontent.com/27012508/52951634-6806dc80-3393-11e9-8082-e79c2337b695.png">
This is what I mean

Reply:
Let me fix it ASAP. Thank you!